core/.bazelrc

```diff
@@ -1,4 +1,5 @@
 build --host_platform=@rules_nixpkgs_core//platforms:host
+build --extra_execution_platforms=@io_tweag_rules_nixpkgs//nixpkgs/platforms:host
```

Comment:
Also, after reverting the commit, CI was all green: https://github.com/tweag/rules_nixpkgs/actions/runs/3095774582

Reply:
iirc back then it failed to pick up the toolchain. i had some longer discussion in slack with someone that lead me to find this was required. maybe things changed since then ?